src/model/utilities/Player.test.js

```diff
@@ -0,0 +1,29 @@
+import player from './Player';
+
+describe('Player', () => {
+  const testPlayer = player();
+  const testQuestion = {
+    answers: ["Declan's men and his cook.", 'Los Pollos driver', 'No-Doze', 'Max Arciniega'],
+    correctAnswer: 'NoDoze',
```

Comment:
Correct answer is not in answers - missing "-"

Reply:
done